LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License
 
-Copyright (c) 2008-2019 Aslak Hellesøy and the Cucumber Team.
+Copyright (c) 2008+ Aslak Hellesøy and the Cucumber Team.
```

Comment:
I don't think you can put an open-ended range here. It's a bummer but we'll have to just update the year every time there's a new release in a new year.

Reply:
I've seen other projects do it? Is there a reason we can't?

Reply:
You need to document the last copyright year so people know when copyright ends. But IANAL.